handlers.go

```diff
@@ -16,8 +16,25 @@ func GetBeers(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
 }
 
 // GetBeer returns a beer from the cellar
-func GetBeer(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
-	// TODO
+func GetBeer(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	w.Header().Set("Content-Type", "application/json")
+
+	ID, err := strconv.Atoi(ps.ByName("id"))
+	if err != nil {
+		w.WriteHeader(http.StatusBadRequest)
+		json.NewEncoder(w).Encode(fmt.Sprintf("%s is not a valid Beer ID, it must be a number.", ps.ByName("id")))
```

Comment:
Another PR reminded me you can also use the shortcut Error func:
`http.Error(w, http.StatusText(http.StatusBadRequest), http.StatusBadRequest)`

I'll change it here to that to match the other endpoints, just thought I'll mention it in case it's useful! :)